esmf_regrid/schemes.py

```diff
@@ -4,9 +4,9 @@
 import copy
 import functools
 
+from cf_units import Unit
```

Comment:
Is this the first time we're importing cf_units? It needs to be added to the requirements.yml files if so.  I expect we get it as a transitive dep from iris, but good to be explicit.